### DISCUSSION TITLE
Public APIの使い方を教えてください！

### DISCUSSION THREAD

**Discussion by UserSuccessTeamLAPRAS** [lapras-inc/lapras-user-community#9] [Show and tell]
LAPRASβプログラムの一環として、[Public API](https://lapras.com/person)をリリースいたしました。
このAPIを使ってアプリなどを作ってみた方、是非シェアしてください！ 🙏 
運営からも宣伝させていただきます 💪

**Comment by kawamataryo** [comment]
公開APIを使って、自分のGitHub Profileにクレジットカード風のLAPRASスコアを掲載できるツールを作りました！
GitHub Actionsのマーケットプレイスで公開しているので簡単に使うことができます。

https://github.com/marketplace/actions/lapras-card-readme

（example）
<a href="https://lapras-card-generator.vercel.app"><img src="https://lapras-card-generator.vercel.app/api/svg?e=4.22&b=3.51&i=2.28&b1=%23020E27&b2=%230E5593&i1=%23030E21&i2=%231688BF&l=ja" width="400"></a>


Generatorを使って自分の好きなデザインでカードをつくることもできるので、是非使ってみてください〜

<img src="https://storage.googleapis.com/zenn-user-upload/deployed-images/51a22f986c94f7ac097cd463.gif">

詳細な設定方法や、実装内容は以下の記事に記載しています！

https://zenn.dev/ryo_kawamata/articles/6e161be042f3d1

**Reply by UserSuccessTeamLAPRAS** [reply]
共有ありがとうございます！
デザインも素敵ですね！
